rust/connlib/tunnel/src/tests.rs

```diff
@@ -184,8 +197,9 @@ impl StateMachineTest for TunnelTest {
             logger,
             relay,
             dns_by_sentinel: Default::default(),
-            gateway_received_icmp_requests: Default::default(),
             client_received_icmp_replies: Default::default(),
+            client_sent_icmp_requests: Default::default(),
+            gateway_received_requests: Default::default(),
```

Comment:
I think a more apt name for this is `outgoing_gateway_requests`, since this is what the gateway is going to send to the resource if I understand correctly